package.json

```diff
@@ -22,11 +22,12 @@
     "prepare": "npm run build",
     "start": "node ./bin/server.js -p 3000",
     "test": "jest",
+    "validate": "componentsjs-compile-config urn:solid-server:my -c config/config-default.json > /dev/null",
```

Comment:
There's the `my` again 😂

Reply:
Ah yeah, that's because #98 is not merged yet.

Reply:
It is now so you can change it 😉

Reply:
Done :)

But it looks like the build fails because LSD is down... :-(

Reply:
Build is succeeding now

Reply:
Yay :) Then this should be good to go once @RubenVerborgh approves.